samples/java-protobuf-eventsourced-customer-registry-subscriber/docker-compose.yml

```diff
@@ -9,10 +9,11 @@ services:
     image: gcr.io/kalix-public/kalix-runtime:1.1.27
     container_name: java-protobuf-eventsourced-customer-registry-subscriber
     ports:
-      - "9001:9000"
+      - "9001:9001"
     extra_hosts:
       - "host.docker.internal:host-gateway"
     environment:
+      HTTP_PORT: "9001"
```

Comment:
README.md still says

> First start the java-protobuf-eventsourced-customer-registry service and proxy. It will run with the default service and proxy ports (8080 and 9000).

Reply:
Ah, sorry, my bad 🤦 everything is good then!